packages/react-native/Libraries/Components/View/View.js

```diff
@@ -23,108 +26,91 @@ export type Props = ViewProps;
-          }
-        : undefined;
-
-    const actualView = (
-      <ViewNativeComponent
-        {...otherProps}
-        accessibilityLiveRegion={
-          ariaLive === 'off' ? 'none' : ariaLive ?? accessibilityLiveRegion
+  const _accessibilityState =
+    accessibilityState != null ||
+    ariaBusy != null ||
+    ariaChecked != null ||
+    ariaDisabled != null ||
+    ariaExpanded != null ||
+    ariaSelected != null
+      ? {
+          busy: ariaBusy ?? accessibilityState?.busy,
+          checked: ariaChecked ?? accessibilityState?.checked,
+          disabled: ariaDisabled ?? accessibilityState?.disabled,
+          expanded: ariaExpanded ?? accessibilityState?.expanded,
+          selected: ariaSelected ?? accessibilityState?.selected,
         }
-        accessibilityLabel={ariaLabel ?? accessibilityLabel}
-        focusable={tabIndex !== undefined ? !tabIndex : focusable}
-        accessibilityState={_accessibilityState}
-        accessibilityElementsHidden={ariaHidden ?? accessibilityElementsHidden}
-        accessibilityLabelledBy={_accessibilityLabelledBy}
-        accessibilityValue={_accessibilityValue}
-        importantForAccessibility={
-          ariaHidden === true
-            ? 'no-hide-descendants'
-            : importantForAccessibility
+      : undefined;
+
+  const _accessibilityValue =
+    accessibilityValue != null ||
+    ariaValueMax != null ||
+    ariaValueMin != null ||
+    ariaValueNow != null ||
+    ariaValueText != null
+      ? {
+          max: ariaValueMax ?? accessibilityValue?.max,
+          min: ariaValueMin ?? accessibilityValue?.min,
+          now: ariaValueNow ?? accessibilityValue?.now,
+          text: ariaValueText ?? accessibilityValue?.text,
         }
-        nativeID={id ?? nativeID}
-        ref={forwardedRef}
-      />
-    );
+      : undefined;
 
-    if (hasTextAncestor) {
-      return (
-        <TextAncestor.Provider value={false}>
-          {actualView}
-        </TextAncestor.Provider>
-      );
-    }
+  const actualView = (
+    <ViewNativeComponent
+      {...otherProps}
+      accessibilityLiveRegion={
+        ariaLive === 'off' ? 'none' : ariaLive ?? accessibilityLiveRegion
+      }
+      accessibilityLabel={ariaLabel ?? accessibilityLabel}
+      focusable={tabIndex !== undefined ? !tabIndex : focusable}
+      accessibilityState={_accessibilityState}
+      accessibilityElementsHidden={ariaHidden ?? accessibilityElementsHidden}
+      accessibilityLabelledBy={_accessibilityLabelledBy}
+      accessibilityValue={_accessibilityValue}
+      importantForAccessibility={
+        ariaHidden === true ? 'no-hide-descendants' : importantForAccessibility
+      }
+      nativeID={id ?? nativeID}
+    />
+  );
 
-    return actualView;
-  },
-);
+  if (hasTextAncestor) {
+    return <TextAncestor value={false}>{actualView}</TextAncestor>;
+  }
 
-View.displayName = 'View';
+  return actualView;
+}
 
 export default View;
```

Comment:
You probably still want to preserve the `component` type.

```suggestion
export default View as component(
  ref?: React.RefSetter<React.ElementRef<typeof ViewNativeComponent>>,
  ...props: ViewProps
);
```